app/views/potential_matches/_no_match_required.html.erb

```diff
@@ -3,7 +3,11 @@
   will be purely random. 
   (You can adjust the assignments by hand afterwards, though!)
   If you want participants matched up on the tags they signed up with, 
-  please update your ") %> 
-  <strong><%= link_to ts("challenge match settings"), 
-    eval("edit_collection_#{challenge_class_name(@collection)}_path(@collection, :anchor => 'match_settings')") %></strong>. 
+  please update your ") %>
+  <strong><%= link_to ts("challenge match settings"),
```

Comment:
Wait, why did this even _have_ the eval? This page is shown only for gift exchanges, not for prompt memes (which don't have matching). You can get rid of the if/else here and just do `edit_collection_gift_exchange_path(@collection, :anchor => 'match_settings')`